final-project/index.html

```diff
@@ -1,14 +1,574 @@
+        let arrow = listItems[i].querySelector('.faq__container--list-item-title-arrow');
+        if(arrow.classList.contains('arrow-up')){
+          arrow.classList.remove('arrow-up');
+          arrow.classList.add('arrow-down');
+        }
+        
+        let title = listItems[i].querySelector('.faq__container--list-item-title');
+        if(title.classList.contains('active-title')){
+          title.classList.remove('active-title');
+        }
+
+        let answer = listItems[i].querySelector('.faq__container--list-item-answer');
+        if(answer.classList.contains('active-faq')){
+          answer.classList.remove('active-faq');
+        }
+      }
+        
+        let arrow = el.querySelector('.faq__container--list-item-title-arrow');
+        if(arrow.classList.contains('arrow-down')){
+          arrow.classList.remove('arrow-down');
+          arrow.classList.add('arrow-up');
+        }else{
+          arrow.classList.remove('arrow-up');
+          arrow.classList.add('arrow-down');
+        }
+        
+        let title = el.querySelector('.faq__container--list-item-title');
+        if(title.classList.contains('active-title')){
+          title.classList.remove('active-title');
+        }else{
+          title.classList.add('active-title');
+        }
+
+        let answer = el.querySelector('.faq__container--list-item-answer');
+        if(answer.classList.contains('active-faq')){
+          answer.classList.remove('active-faq');
+        }else{
+          answer.classList.add('active-faq');
+        }
+    }
+    window.onload = function () {
+      var carousel = new Carourel({klass: '.customer__container-image', prefix: 'customer'});
+      window.addEventListener("resize", onResizeCustomers);
+      onResizeCustomers();
+      function onResizeCustomers() {
+        var wh = window.innerWidth;
+
+        if (wh <= 415 && !carousel.isStarted()) {
+          carousel.init();
+        } else if (wh > 415 && carousel.isStarted) {
+          carousel.stopInterval();
+        }
+      }
+
+      var quotesCarousel = new Carourel({klass: '.quotes__user--content-carousel-card', prefix: 'quotes'});
+      quotesCarousel.init();
+
+      var url = "https://wt-4662f45b9eefda7172b747b28d23efdb-0.sandbox.auth0-extend.com/blog";
+      fetch(url)
+        .then(function (data) {
+          return data.json();
+        })
+        .then(response => {
+          buildResourceItems(response);
+          var carouselResources = new Carourel({klass: '.resources__container-item', prefix: 'resources', withArrows: true});
+          carouselResources.init();
+        })
+        .catch(function (error) {
+          console.log('error', error);
+        });
+
+      function buildResourceItems(data) {
+        for(let i = 0; i < data.articles.length; i ++) {
+          let dataArt = data.articles[i];
+
+          let baseElm = document.querySelector('.resources__container');
+
+          let itemElm = document.createElement('div');
```

Comment:
Probably it would have been more readable and maintainable to have these divs referenced and transformed from the HMTL instead of creating the whole section with JS. In the case of hiding the articles HTML when the browser doesn't have JS, it could be `display:none` by default.